README.md

```diff
@@ -12,29 +12,52 @@
 
 <!--te-->
 
```

Comment:
Maybe we can add a link to our website in the about section?
![Selection_154](https://user-images.githubusercontent.com/11608884/94586867-74029200-028a-11eb-9925-2290a85814c0.png)